---
title: Continuation.resumeWithException - arrow-core
---

[arrow-core](../../index.html) / [arrow.core](../index.html) / [Continuation](index.html) / [resumeWithException](./resume-with-exception.html)

# resumeWithException

`abstract fun resumeWithException(exception: `[`Throwable`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-throwable/index.html)`): `[`Unit`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/-unit/index.html)